resources/views/components/menu/brand.blade.php

```diff
@@ -1,9 +1,3 @@
-<a class="navbar-brand {{ request()->routeIs('home') ? 'lg:hidden' : '' }}"
-   href="{{ route('home') }}">
-    @if(request()->cookie('logo') === 'retro')
-        <img class="hidden md:inline-block" src="{{ asset('assets/images/ra-logo-sm.webp') }}" alt="{{ config('app.name') }}">
-        <img class="inline-block md:hidden" src="{{ asset('assets/images/ra-icon.webp') }}" alt="{{ config('app.name') }}">
-    @else
-        <img class="h-6" src="{{ asset('assets/images/ra-icon.webp') }}" alt="{{ config('app.name') }}">
-    @endif
+<a class="ml-3 {{ request()->routeIs('home') ? 'lg:hidden' : '' }}" href="{{ route('home') }}">
+    <img class="h-6" src="{{ asset('assets/images/ra-icon.webp') }}" alt="{{ config('app.name') }}">
```

Comment:
I'm not sure how well this comes across in an image, but if you scroll down on the main page so the navbar is docked to the top, then go to some other page, the menu elements shift down a pixel (but not the red dots)

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/a62fdae4-e7d3-4cc0-8ad6-e1cbcbb37f5d)

I assume the icon is causing the height of the navbar to change a little.

Reply:
What an interesting find! Turned out that rounding errors in the top area texts' line heights cause the height of the top area to shift ever so slighty, affecting the whole page.
Solved by setting a fixed height for the top brand area: https://github.com/RetroAchievements/RAWeb/pull/1831/commits/954dfefbcebd17b9d4149ccf4a77973098d6da15